src/ONYXKEYS.ts

```diff
@@ -1199,6 +1202,7 @@ type OnyxValuesMapping = {
     [ONYXKEYS.NVP_LAST_ANDROID_LOGIN]: string;
     [ONYXKEYS.TRANSACTION_THREAD_NAVIGATION_REPORT_IDS]: string[];
     [ONYXKEYS.HYBRID_APP]: OnyxTypes.HybridApp;
+    [ONYXKEYS.NVP_DRAFT_REPORT_COMMENTS]: OnyxTypes.DraftReportComments;
```

Comment:
Let's be sure to mark the old key as deprecated

Reply:
Actually, thinking about this some more, is there any reason we can't fully remove the old key in this PR, since draft comments (until now) have been front-end-only data?